src/functions-and-arrays.js

```diff
@@ -1,41 +1,118 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1
+  }
+  else return num2
+}
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(words) {
+  let longestWord = words[0];
+  let result;
+  if (words.length === 0) {
+    return null
+  }
```

Comment:
For shorter syntax, if you return only on line after if statement, you could write it like this:
```
if (words.length === 0) return null;
```